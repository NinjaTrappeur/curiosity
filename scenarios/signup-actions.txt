reset
queues
user create alice secret alice@example.com --accept-tos
queues
step --all --dry
step --all
queues
