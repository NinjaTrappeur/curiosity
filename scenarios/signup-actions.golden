1: reset
Resetting to the empty state.
2: queues
Email addresses to verify:
Emails to send:
3: user signup alice secret alice@example.com --accept-tos
User created: USER-1
4: queues
Email addresses to verify:
USER-1 alice
Emails to send:
EMAIL-1 alice@example.com
5: step --all --dry
Setting user email addr. to verified: alice@example.com
Sending email: EMAIL-1
All steps done.
6: step --all
Setting user email addr. to verified: alice@example.com
Sending email: EMAIL-1
All steps done.
7: queues
Email addresses to verify:
Emails to send:
